src/content/1.7/development/Multistore/shop-context/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Shops context
+menuTitle: Shops context
+summary: "How to get the current shop context"
+weight: 20
+---
+
+# Shop context
+
+In many cases, your code will need to know if the multistore mode is enabled, if there are more than one shops activated, and if yes, what is the current shop context. This is where the shop context and multistore feature adapters will come in handy.
+
+## The multistore feature adapter
+
+**Service identifier:** `prestashop.adapter.feature.multistore`  (it takes the legacy configuration adapter as a parameter) <br>
+**Class:** `PrestaShop\PrestaShop\Adapter\Feature\MultistoreFeature`
+
+
+The multistore feature adapter can be used to:
+
+- detect if multistore was enabled in the configuration
+
+- detect if multistore is active (if it is enabled, and if there are more than one active shops)
+
+- enable/disable the multistore feature
+
+Example:
+
+```php
+<?php
+
+// get the multistore feature adapter from the service container
+$multistoreFeature = $this->get('prestashop.adapter.feature.multistore');
+
+// Is the multistore feature enabled ?
+$isMultistoreEnabled = $multistoreFeature->isActive();
+
+// This will check that multistore is enabled AND that there are at least two active shops
+$isMultistoreUsed = $multistoreFeature->isUsed();
+
+// Disable / enable multistore feature
+$multistoreFeature->disable();
+$multistoreFeature->enable();
+
+```
+
+## The shop context adapter
+
+**Service identifier:** `prestashop.adapter.shop.context` <br>
+**Class:** `PrestaShop\PrestaShop\Adapter\Shop\Context`
+
+When the multistore feature is enabled, at the top of all the back office pages, a dropdown appears, it allows selecting the shop context we want (all shops, a group shop, a single shop...). The shop context adapter will tell you in what context your are operating.
```

Comment:
```suggestion
When the multistore feature is enabled, at the top of all the back office pages, a dropdown appears, it allows selecting the shop context we want (all shops, a group shop, a single shop...). The shop context adapter will tell you in what context your shop is operating.
```

Reply:
I understand that you imply that when we are in multistore, each store belongs to a shop?  
I find it a bit ambiguous, what we want to know here is in which shop context the code is operating. 

I could replace "you are operating" by "your code is operating" though